zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -194,7 +194,12 @@ public boolean accept(File pathname) {
 
     try {
       out.reset();
-      npmCommand("install --loglevel=error");
+      String npmCommand =
+              "install --fetch-retries=2 --fetch-retry-factor=1 " +
```

Comment:
One question.
The npm command which is for install that @1ambda said above, did execute `npm install *some artifact* npm install --loglevel=error` without error?

https://github.com/NohSeho/zeppelin/blob/b7296ca0d60ba99d6c874831330a823ccea2f2b0/zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java#L380-#L382

I think, it should be removed. because there's duplicated command.